### PR TITLE
Issue #304: Rename the 'main menu' menu to 'primary navigation'.

### PR DESCRIPTION
**Pull request by rocketnova**
PR for https://github.com/backdrop/backdrop-issues/issues/304.

### CODE REVIEW

File: core/modules/menu/menu.install [hunk 1]
```diff
@@ -47,7 +47,7 @@ function menu_install() {
   $system_menus = menu_list_system_menus();
   $t = get_t();
   $descriptions = array(
-    'main-menu' => $t('The <em>Main</em> menu is used on many sites to show the major sections of the site, often in a top navigation bar.'),
+    'main-menu' => $t('The <em>Primary</em> navigation is used on many sites to show the major sections of the site, often in a top navigation bar.'),
```
Comment (jenlampton) [inline core/modules/menu/menu.install]: I think navigation should be inside the em tag here too, how about something like...

'The <em>Primary navigation</em> menu is used on many sites to show the major sections of the site, often in a top navigation bar.'

**Comment by jenlampton** [comment]
One change as noted above, otherwise looking great! :)

**Comment by rocketnova** [comment]
Change made!

**Comment by jenlampton** [comment]
This looks great, I'm not sure why tests are failing. Perhaps we need a rebase?

**Comment by jenlampton** [comment]
Rebased and created a new PR over at https://github.com/backdrop/backdrop/pull/528

**Pull request closed without merging**